tests/ProcessMaker/Models/CallActivity.php

```diff
@@ -35,7 +36,9 @@ function ($self, TokenInterface $token) {
                     ProcessInterface::EVENT_PROCESS_COMPLETED,
                     function ($self, $closedInstance) use($token, $instance) {
                         if ($closedInstance === $instance) {
-                            $token->setStatus(ActivityInterface::TOKEN_STATE_COMPLETED);
+                            if ($token->getStatus() !== ActivityInterface::TOKEN_STATE_FAILING) {
```

Comment:
Added validation for falling scenario in call activity required because the token are now persistent